cart/src/test/java/com/yas/cart/service/CartServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package com.yas.cart.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.yas.cart.model.Cart;
+import com.yas.cart.repository.CartItemRepository;
+import com.yas.cart.repository.CartRepository;
+import com.yas.cart.viewmodel.CartItemVm;
+import com.yas.commonlibrary.exception.BadRequestException;
+import com.yas.commonlibrary.exception.NotFoundException;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CartServiceTest {
+
+    private CartService cartService;
+    private CartRepository cartRepository;
+    private CartItemRepository cartItemRepository;
+    private ProductService productService;
+
+    @BeforeEach
+    public void setUp() {
+        cartRepository = mock(CartRepository.class);
+        cartItemRepository = mock(CartItemRepository.class);
+        productService = mock(ProductService.class);
+        cartService = new CartService(cartRepository, cartItemRepository, productService);
```

Comment:
Utilize @InjectMocks and @Mock annotation instead of using mock() explicitly.

Reply:
I’ve updated the code to utilize @InjectMocks and @Mock annotations. Please take a review, thank you.